src/modules/Security.rb

```diff
@@ -688,6 +708,11 @@ def Write
       true
     end
 
+    SYSCTL_VALUES = {
```

Comment:
Good question, the constants are normally written at top but in Yast i had seen big classes with a lot of lines, and had seen that there is some ones that are written constants declarations just above the method using it.

So in case it will be used in different parts of the code i move it to the top, but in this case you don't have to scroll up to read it. So... what is the preferred way? for me make sense both alternatives